app/models/health_card.rb

```diff
@@ -1,16 +1,28 @@
 class HealthCard
```

Comment:
Reece had mentioned not creating a new HealthCard class to avoid confusion with the HealthCard class from our library. Would this be a part of a separate PR?

Reply:
Yeah, I originally intended to do that, but after looking at it for a while I think we need to have discussion about it.